.github/workflows/trigger-by-label.yaml

```
@@ -0,0 +1,24 @@
+name: Trigger workflow by label
+
+on:
+  workflow_call:
+
+jobs:
+  trigger:
+    name: Trigger workflow and remove label
```

This isn't technically triggering the Github workflow. It removes the label and its success serves as a trigger for another job on the consumer side. Maybe it should just indicate what it does:
```suggestion
jobs:
  remove-label:
    name: Remove label
```

Same for the workflow name and file name.